### PR TITLE
get rid of references to 'hpcugent' organisation after move to github.com/easybuilders on website

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@easybuilders/easybuild-docs-maintainers please review/merge?

**Comment by damianam** [comment]
`bootstrap_eb.py`, `fosdem14.html` and `params.json`(I guess this last one is autogenerated) have the old references. Is there any reason to keep them?

**Comment by boegel** [comment]
@damianam the `bootstrap.py` should be updated after https://github.com/easybuilders/easybuild-framework/pull/2256 is merged

I'll change the references in `fosdem14.html`, although I'm not sure if it's of much use anymore (but keeping it around is fine too, and 'safer').

The `params.json` is generated by GitHub itself, I'm not going to touch that.

File: index.html [hunk 1]
```diff
@@ -35,17 +35,18 @@
 
 <div class=news>
 <ul>
+<li><em>20170705</em> - EasyBuild repositories have been migration to <a href="https://github.com/easybuilders">https://github.com/easybuilders</a></li>
```
Comment (damianam) [inline index.html]: "have been migration"?

Comment (boegel) [inline index.html]: Nice catch!

**Comment by damianam** [comment]
Mmmmh, none of the changed (or the old, for that matter) urls in `fosdem14.html` work :-?

**Comment by boegel** [comment]
@damianam fixed, should be `*.github.io`, apparently `.com` only works for old sites...

**Comment by damianam** [comment]
lgtm. Going in! Thanks @boegel! (am I doing this right?)

**Comment by boegel** [comment]
@damianam You can check yourself by visiting http://easybuilders.github.io/easybuild/ ;)

**Pull request merged**